### ISSUE TITLE
`semantic-release changelog` outputs nothing

### ISSUE THREAD

**Issue by halkeye** [relekang/python-semantic-release#67]
There's no getting started guide that I've found. It doesn't tell you if you did anything wrong, just fails.

Due to some docs and trial and error I have this flow:
```
cat << EOF > setup.cfg
[semantic_release]
upload_to_pypi=False
version_variable=version.py:__version__
EOF

cat << EOF > version.py
__version__ = '0.0.1'
EOF

git init
git add .
git commit -m "feat(*): Does all the things"
curl https://www.gitignore.io/api/python > .gitignore
pip install python-semantic-release
semantic-release changelog
```

**Comment by halkeye** [comment]
Publish fails, probably due to a lack of tag?
```halkeye@blackpanther:~/git/it-jira-bamboohr/a/i_like_pie$ semantic-release publish
Current version: 0.0.1
Traceback (most recent call last):
  File "/home/halkeye/.virtualenvs/it-jira-bamboohr/bin/semantic-release", line 11, in <module>
    sys.exit(main())
  File "/home/halkeye/.virtualenvs/it-jira-bamboohr/local/lib/python2.7/site-packages/click/core.py", line 610, in __call__
    return self.main(*args, **kwargs)
  File "/home/halkeye/.virtualenvs/it-jira-bamboohr/local/lib/python2.7/site-packages/click/core.py", line 590, in main
    rv = self.invoke(ctx)
  File "/home/halkeye/.virtualenvs/it-jira-bamboohr/local/lib/python2.7/site-packages/click/core.py", line 936, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/halkeye/.virtualenvs/it-jira-bamboohr/local/lib/python2.7/site-packages/click/core.py", line 782, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/halkeye/.virtualenvs/it-jira-bamboohr/local/lib/python2.7/site-packages/click/core.py", line 416, in invoke
    return callback(*args, **kwargs)
  File "/home/halkeye/.virtualenvs/it-jira-bamboohr/local/lib/python2.7/site-packages/semantic_release/cli.py", line 171, in cmd_publish
    return publish(**kwargs)
  File "/home/halkeye/.virtualenvs/it-jira-bamboohr/local/lib/python2.7/site-packages/semantic_release/cli.py", line 114, in publish
    level_bump = evaluate_version_bump(current_version, kwargs['force_level'])
  File "/home/halkeye/.virtualenvs/it-jira-bamboohr/local/lib/python2.7/site-packages/semantic_release/history/logs.py", line 35, in evaluate_version_bump
    for _hash, commit_message in get_commit_log('v{0}'.format(current_version)):
  File "/home/halkeye/.virtualenvs/it-jira-bamboohr/local/lib/python2.7/site-packages/semantic_release/vcs_helpers.py", line 18, in get_commit_log
    for commit in repo.iter_commits(rev):
  File "/home/halkeye/.virtualenvs/it-jira-bamboohr/local/lib/python2.7/site-packages/git/objects/commit.py", line 265, in _iter_from_process_or_stream
    finalize_process(proc_or_stream)
  File "/home/halkeye/.virtualenvs/it-jira-bamboohr/local/lib/python2.7/site-packages/git/util.py", line 154, in finalize_process
    proc.wait()
  File "/home/halkeye/.virtualenvs/it-jira-bamboohr/local/lib/python2.7/site-packages/git/cmd.py", line 309, in wait
    raise GitCommandError(self.args, status, self.proc.stderr.read())
git.exc.GitCommandError: 'git rev-list ...v0.0.1 --' returned with exit code 128
stderr: 'fatal: bad revision '...v0.0.1'```

**Comment by halkeye** [comment]
same with version

**Comment by halkeye** [comment]
```
git tag v0.0.1
git remote add origin git@github.com:halkeye/test_test_test_test.git
git push -u origin master
halkeye@blackpanther:~/git/it-jira-bamboohr/a/i_like_pie$ semantic-release publish
Current version: 0.0.1
Creating new version..
Current version: 0.0.1
No release will be made.
Version failed, no release will be published.
```

**Comment by halkeye** [comment]
```
date > date
git add date
git commit -m "feat(date): I love dates"
semantic-release publish
```

^^^ would have worked, but i entered the wrong GH_TOKEN so it failed, now I can't publish though I can changelog.

I'm thinking part of the problem might be my "feat(*): feature" message?

**Comment by halkeye** [comment]
I tried adding a few more commits:
```
...
git commit -m "feat(*): dates are chewie"
...
git commit -m "feat: dates are chewie"
```

but changelog just shows the initial release..

I've found now that I can publish again, and changelog shows the last release.

Does that sorta describe the flow? commit, publish, changelog?

**Comment by halkeye** [comment]
I will continue to poke at this to see if i can't make it more user friendly

#68 helps a bit

**Comment by relekang** [comment]
@halkeye Is this fixed by #68?

**Comment by halkeye** [comment]
I think its safe to close. I'm having a hard time setting up the reproduction steps quickly enough to test it.
